source/codegen/metadata/nidmm/attributes.py

```diff
@@ -1,645 +1,666 @@
         'resettable': False,
         'type': 'ViReal64'
     },
     1250301: {
-        'access': 'read-write',
-        'channel_based': False,
-        'enum': 'SampleCount',
+        'codegen_method': 'public',
+        'grpc_type': 'sint32',
         'name': 'SAMPLE_COUNT',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250302: {
-        'access': 'read-write',
-        'channel_based': False,
+        'codegen_method': 'public',
         'enum': 'SampleTrigger',
+        'grpc_type': 'sint32',
         'name': 'SAMPLE_TRIGGER',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250303: {
-        'access': 'read-write',
-        'attribute_class': 'AttributeViReal64TimeDeltaSeconds',
-        'channel_based': False,
-        'enum': 'SampleInterval',
+        'codegen_method': 'public',
+        'grpc_type': 'double',
         'name': 'SAMPLE_INTERVAL',
         'resettable': False,
-        'type': 'ViReal64',
-        'type_in_documentation': 'hightime.timedelta, datetime.timedelta, or float in seconds'
+        'type': 'ViReal64'
     },
     1250304: {
-        'access': 'read-write',
-        'channel_based': False,
-        'enum': 'TriggerCount',
+        'codegen_method': 'public',
+        'grpc_type': 'sint32',
         'name': 'TRIGGER_COUNT',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250305: {
-        'access': 'read-write',
-        'channel_based': False,
+        'codegen_method': 'public',
         'enum': 'MeasurementCompleteDest',
+        'grpc_type': 'sint32',
         'name': 'MEAS_COMPLETE_DEST',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250321: {
-        'access': 'read-write',
-        'channel_based': False,
-        'enum': 'ApertureTime',
+        'codegen_method': 'public',
+        'grpc_type': 'double',
         'name': 'APERTURE_TIME',
         'resettable': False,
         'type': 'ViReal64'
     },
     1250322: {
-        'access': 'read-write',
-        'channel_based': False,
+        'codegen_method': 'public',
         'enum': 'ApertureTimeUnits',
+        'grpc_type': 'sint32',
         'name': 'APERTURE_TIME_UNITS',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250331: {
-        'access': 'read only',
-        'channel_based': False,
+        'codegen_method': 'public',
+        'grpc_type': 'double',
         'name': 'AUTO_RANGE_VALUE',
         'resettable': False,
         'type': 'ViReal64'
     },
     1250332: {
-        'access': 'read-write',
-        'channel_based': False,
+        'codegen_method': 'public',
         'enum': 'AutoZero',
+        'grpc_type': 'sint32',
         'name': 'AUTO_ZERO',
         'resettable': False,
         'type': 'ViInt32'
     },
     1250333: {
-        'access': 'read-write',
-        'channel_based': False,
-        'enum': 'PowerLineFrequencies',
+        'codegen_method': 'public',
+        'enum': 'PowerlineFrequency',
```

Comment:
Enum name changed... why?

Reply:
There was already a similar enum in the base hapigen metadata and referenced by this attribute. I think I'll go with this.